### PR TITLE
FEATURE: Visible tree deterioration

### PR DESCRIPTION
**Pull request by jwiggins**
You can eat the tree roots now. But the consequences are dire...

### CODE REVIEW

**Review by fjebaker — APPROVED** [review]
Sehr gut

File: cart/world.go [hunk 1]
```diff
@@ -105,6 +128,22 @@ func (world *World) Update(t int32, player *Player, game *Game) {
...
+func treeLifeDetails(life int8) (byte, byte, byte) {
+	switch life {
+	case 3:
+		return 16, 31, 128
+	case 2:
+		return 144, 159, 16
+	case 1:
+		return 160, 175, 16
+	case 0:
+		return 176, 191, 0
```
Comment (jwiggins) [inline cart/world.go]: We can update this to give the tree a longer life

Comment (fjebaker) [reply cart/world.go]: You can have named return variables in Go with
```go
func foo() (a int32, b int32, c int32) {
    a = 10
    b = 20
    c = 30
    return
}
```
Can we maybe use that here just to give a bit more meaning to the triple?

Comment (jwiggins) [reply cart/world.go]: Done

**Pull request merged**